.gitignore

```diff
@@ -38,6 +38,7 @@ test/truffle/cexts/**/mkmf.log
 
 # Maven
 lib/truffleruby.jar
+target
```

Comment:
Ok will change. But are most of the rest wrong as well then? `lib/truffleruby.jar` for example?